packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -844,22 +848,36 @@ class IntelliJValidatorOnMac extends IntelliJValidator {
     return validators;
   }
 
+  String get plistFile {
+    _plistFile ??= globals.fs.path.join(installPath, 'Contents', 'Info.plist');
+    return _plistFile;
+  }
+  String _plistFile;
+
   @override
   String get version {
-    if (_version == null) {
-      final String plistFile = globals.fs.path.join(installPath, 'Contents', 'Info.plist');
-      _version = PlistParser.instance.getValueFromFile(
+    _version ??= PlistParser.instance.getValueFromFile(
         plistFile,
         PlistParser.kCFBundleShortVersionStringKey,
       ) ?? 'unknown';
-    }
     return _version;
   }
   String _version;
 
   @override
   String get pluginsPath {
+    final String altLocation = PlistParser.instance.getValueFromFile(plistFile, 'JetBrainsToolboxApp');
```

Comment:
Ah, sorry for being unclear. I meant that you should cache the plugins path after reading and parsing the plist file, like:
```dart
@override
String _pluginsPath;
String get pluginsPath {
  if (_pluginsPath != null) {
    return _pluginsPath;
  }
  final String altLocation = PlistParser.instance.getValueFromFile(plistFile, 'JetBrainsToolboxApp');
  ...
}
```